### DISCUSSION TITLE
Filter and sort icon

### DISCUSSION THREAD

**Discussion by nowshads** [KevinVandy/material-react-table#725] [Q&A] [answered]
1. How can sort icon visible always instead of hover?
2. In filter how can i get the date picker, where for select and input is available
3. How can i add an extra icon next to column name in header

**Comment by KevinVandy** [comment, marked as answer]
MRT v2 just entered beta with the release of v2.0.0-beta.0. Going forward it will show the sort icons in a low opacity by default

<img width="633" alt="image" src="https://github.com/KevinVandy/material-react-table/assets/28243511/92ee0009-b951-430e-b395-8f49a3d2f6b0">

Also in v2, date and date-range filter variants are built-in.

You can always use a custom `Header` render to customize and add buttons to the header.

**Reply by nowshads** [reply]
When i add with Header, header name does't display in header show/hide and on click on that icon sorting is happening
